docs/user/zvm/install_upi.md

```diff
@@ -253,14 +253,61 @@ INFO Waiting up to 30m0s for the bootstrap-complete event...
 The Cluster Image Registry [Operator][cluster-image-registry-operator] does not pick an storage backend for `None` platform. Therefore, the cluster operator will be stuck in progressing because it is waiting for the administrator to [configure][cluster-image-registry-operator-configuration] a storage backend for the image-registry.
 [NFS][openshift-nfs] should be picked as a [storage-backend][nfs-storage-backend].
 
-TODO - document NFS backend setup.
 
-Alternatively, for non-production clusters, emptyDir can be used for testing.
+#### Configuring NFS
+
+NFS is currently the only supported persistent storage option for OpenShift on Z.  To make an existing NFS share accessible for OpenShift to use as persistent storage, users must first attach it as a Persistent Volume.  At least 100GB of NFS storage space must be available for the image registry claim.
+
+```apiVersion: v1
```

Comment:
This needs a line break:

    ```
    apiVersion: v1

Otherwise, Markdown/CommonMark treat that text as part of the info string and not inside the code fence (it gets omitted when rendered).